docs/data/tree-view/tree-item-customization/tree-item-customization.md

```diff
@@ -19,7 +19,7 @@ Hover over them in the demo below to see each slot.
 <!-- TBD which option is the best: interactive or image -->
 <!-- {{"demo": "CustomTreeItemDemo.js", "hideToolbar": true}} -->
 
-{{"component": "modules/components/TreeItem2Anatomy.js"}}
+{{"component": "modules/components/TreeItemAnatomy.js"}}
```

Comment:
The labels on the image still have the old naming. I'm creating a follow-up to replace them 😄